### PR TITLE
added blog.js file

### PR DESCRIPTION
**Pull request by kwekuadaboh**
I have add a new bare bone js file with just console.log('') statement
please review and pull --- thanks
[https://stackoverflow.com/questions/42167345/git-set-local-user-name-and-user-email-different-for-each-repo](url)

### CODE REVIEW

**Comment by adaboh** [comment]
alright 
I will go through and provide you with the feedback

**Comment by kwekuadaboh** [comment]
thanks 
[https://stackoverflow.com/questions/29971053/how-to-open-visual-studio-code-from-the-command-line-on-osx](url)

**Comment by kwekuadaboh** [comment]
[https://stackoverflow.com/questions/29971053/how-to-open-visual-studio-code-from-the-command-line-on-osx]

**Comment by kwekuadaboh** [comment]
https://stackoverflow.com/questions/29971053/how-to-open-visual-studio-code-from-the-command-line-on-osx

File: blog.html [hunk 1]
```diff
@@ -18,5 +18,7 @@ <h1>Blog initaition </h1>
       <li>BEM</li>
       <li>sass css</li>
     </ul>
+
+    <script src="blog.js" charset="utf-8"></script>
```
Comment (adaboh) [inline blog.html]: rename this app.js

Comment (adaboh) [inline blog.html]: Rename this app.js

Comment (kwekuadaboh) [reply blog.html]: alright 
done

Comment (kwekuadaboh) [reply blog.html]: kk

**Review by adaboh — CHANGES_REQUESTED** [review]
add another script file

**Pull request closed without merging**